packages/web/src/components/add-to-playlist/desktop/AddToPlaylistModal.tsx

```diff
@@ -68,18 +71,46 @@ const AddToPlaylistModal = () => {
     )
   }, [searchValue, account, currentCollectionId])
 
+  const playlistTrackIdMap: Record<number, number[]> = filteredPlaylists.reduce(
```

Comment:
same potential thing with useMemo here? i think type wise, lets do `.reduce<Record<number, number[]>>` and then i think the typing will work out there?

Reply:
i think it's also not the end of the world to just do `track_ids.some(track_id => track_id === t.track)` so we dont have to construct an object